pyproject.toml

```diff
@@ -23,7 +23,7 @@ repository = "https://github.com/z3z1ma/dbt-osmosis"
 [tool.poetry.dependencies]
 python = ">=3.9,<3.9.7 || >3.9.7,<3.12"
 click = ">7"
-dbt-core = ">=1.6,<1.9"
+dbt-core = ">=1.8,<1.10"
```

Comment:
Looking at dbt-adapters, it seems that the next version will be 1.10.x, not 2.0.0.

- https://github.com/dbt-labs/dbt-adapters/releases/tag/v1.10.4